src/unit_tests/analysisd/test_decoder_syscollector.c

```diff
@@ -1694,6 +1759,35 @@ void test_syscollector_dbsync_network_address_valid_msg_inserted(void **state)
     const char *result = "ok ";
     int sock = 1;
 
+    expect_any(__wrap_wdbc_query_ex, *sock);
+    expect_string(__wrap_wdbc_query_ex, query, query);
+    expect_any(__wrap_wdbc_query_ex, len);
+    will_return(__wrap_wdbc_query_ex, result);
+    will_return(__wrap_wdbc_query_ex, 0);
+    expect_string(__wrap__mdebug2, formatted_msg, "Field 'proto' cannot be obtained.");
+    expect_string(__wrap__mdebug2, formatted_msg, "Error while mapping 'proto' field value.");
+
+    int ret = DecodeSyscollector(lf, &sock);
+
+    assert_int_not_equal(ret, 0);
+}
+void test_syscollector_dbsync_network_address_valid_msg_inserted(void **state)
+{
```

Comment:
It would be nice to add some spaces and maybe some comments to separate tests from different components (hardware, network, etc)